storage-providers/filecoin-deals/filecoin-programs.md

```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
 
-### Filswan
+### Swan (Filswan)
+
+[![GitHub Org's stars](https://img.shields.io/github/stars/filswan)](https://github.com/filswan)
+[![Read the doc](https://img.shields.io/badge/docs-filswan.com-blue)](https://docs.filswan.com/)
+[![Join Slack](https://img.shields.io/badge/join-Slack-purple)](https://filecoinproject.slack.com/archives/C04LBAMBDPD)
+
+[Swan](https://www.filswan.com/homepage) is a leading provider of cross-chain cloud computing solutions. Its suite of tools revolutionizes how developers access resources across multiple chains.
+
+Swan Cloud provides decentralized cloud computing solutions for Web3 projects by integrating storage, computing, and payment into one suite.
+
+The Swan Web3 suite includes:
+
+#### Swan Client
+
+[![GitHub Org's stars](https://img.shields.io/github/stars/filswan/go-swan-client)](https://github.com/filswan/go-swan-client)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/go-swan-client)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/go-swan-client/README.md)
+
+Swan-client is an important Web3 toolkit. It provides different tools to help users connect to the Web3 world.
+
+#### Swan Provider
+
+[![Github repo stars](https://img.shields.io/github/stars/filswan/go-swan-provider)](https://github.com/filswan/go-swan-provider)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/go-swan-provider)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/go-swan-provider/README.md)
+
+A golang implementation for Swan Service provider.
+
+#### Multichain Storage
+
+[![Github repo stars](https://img.shields.io/github/stars/filswan/multi-chain-storage)](https://github.com/filswan/multi-chain-storage)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/multi-chain-storage)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/multi-chain-storage/README.md)
+
+[Multichain Storage](https://multichain.storage/home) is a smart-contract-based cross-chain storage gateway integrated with Oracle technology and the Filecoin networks. It accelerates the mass adoption of decentralized storage by bridging multiple blockchain networks.
+
+#### FS3
+
+[![Github repo stars](https://img.shields.io/github/stars/filswan/fs3)](https://github.com/filswan/fs3)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/fs3)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/fs3/README.md)
+
+A high Performance, Kubernetes Native Object Storage.
+
+## Deprecated
```

Comment:
I can't remember what I said here - something like 'Former tools and programs' ?

Reply:
Yes it's changed on the next push.